app/src/main/java/tech/relaycorp/gateway/background/ConnectionState.kt

```diff
@@ -2,8 +2,8 @@ package tech.relaycorp.gateway.background
 
 sealed class ConnectionState {
     object Disconnected : ConnectionState()
-    object InternetWithoutPublicGateway : ConnectionState()
-    object InternetAndPublicGateway : ConnectionState()
+    object InternetWithoutInternetGateway : ConnectionState()
+    object InternetAndGateway : ConnectionState()
```

Comment:
The second name isn't consistent with the first one. How about `InternetWithoutGateway` + `InternetWithGateway`, for example?